src/rj_strategy/src/coordinator/waller.cpp

```diff
@@ -0,0 +1,48 @@
+#include "rj_strategy/coordinator/waller.hpp"
+
+namespace strategy {
+
+Waller::Waller() : Coordinator("waller_srv", "waller_data", "waller_node") {
+    walling_robots_.fill(-1);
```

Comment:
```suggestion
    walling_robots_.fill(std::numeric_limits<uint8_t>::max());
```
I would prefer not to have overflow and make this explicit